library/src/main/java/com/squareup/timessquare/CalendarCellView.java

```diff
@@ -30,16 +30,23 @@
   private static final int[] STATE_RANGE_LAST = {
       R.attr.tsquare_state_range_last
   };
-
   private static final int[] STATE_BLOCKED = {
       R.attr.tsquare_state_blocked
   };
+  private static final int[] STATE_SURFING = {
+      R.attr.tsquare_state_surfing
+  };
+  private static final int[] STATE_HOSTING = {
+      R.attr.tsquare_state_hosting
+  };
```

Comment:
Shouldn't we need `STATE_SURFING_FIRST`, `STATE_SURFING_MIDDLE`, `STATE_SURFING_LAST` like 'STATE_RANGE'? This would allow to add specific UI to begining middle and end like half rounded background or other.